roles/manage-user-access/tasks/sshd-update.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: "Update ssh to restrict root password access" 
+  lineinfile:
```

Comment:
Consider using the ansible `ini_file` module instead (with the `section` parameter set to null)
http://docs.ansible.com/ansible/latest/ini_file_module.html